packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,265 @@
+# Card Decisions <!-- omit in toc -->
+
+Cards are used to group similar concepts and tasks together to make easier for merchants to scan, read, and get things done. In simpler words Cards help seprates content into sections. They are the surfaces that display content and actions on a single topic. They should be easy to scan for relevant and actionable information.
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [Card Requirements](#card-requirements)
+- [Card API Constraints](#card-api-constraints)
+- [API](#api)
+  - [`Card` API](#card-api)
+  - [`Card.Header` API](#cardheader-api)
+  - [`Card.Footer` API](#cardfooter-api)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=5200%3A0) to all variants of the Card component
+
+## Anatomy
+
+<img src="./card-anatomy.png" width="80%" />
+
+Components: 
+
+- `Card`
+- `Card.Header`
+- `Card.Body`
+- `Card.Footer`
+
+## Card Requirements
+
+- 2 Levels (color levels)
+- Slot for user's content
+- Have Header/Footer
+- **Header**
+  - Header have leading and trailing sections
+  - Leading section can have: 
+    - Icon
+    - Title
+    - Subtitle
+    - Counter
+  - Trailing section can have:
+    - Badge
+    - Text
+    - Link
+    - Icon Button
+- **Footer**
+  - Footer have leading and trailing sections:
+  - Leading section can have: 
+    - Title
+    - Subtitle
+  - Trailing section can have:
+    - Primary action button
+    - Secondary action button
+
+
+## Card API Constraints 
+
+**Header Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+- `Icon`
+  - Size & color cannot be changed
+- `Counter`
+  - Size can be changed but not recommended
+  - Variant can be changed
+
+**Header Trailing Section**
+
+Anything can be changed in these components, but whats given out of the box are the recommended: 
+
+Only one of the component can be used at a time: 
+- Badge
+- Text
+- Link
+- IconButton - tertiary
+
+Other recommendations from design side: 
+
+- Never use a primary/secondary action in the header, footer are meant for such actions.
+  - This means it's not recommended to change variant of IconButton
+
+**Footer Leading Section**
+
+- `Title` & `Subtitle`
+  - Size & text variant cannot be changed
+
+**Footer Trailing Sections**
+
+- Only two action buttons can be present `primary` or `secondary`
+- Users can totally omit action buttons
+- They can also use either variants of buttons `primary` or `secondary`
+
+## API
+
+With the above constraints we can propose the Card API as follows: 
+
+Sample usage:
+
+```jsx
+<Card backgroundLevel={1}>
+  <Card.Header 
+    title="Payments Links" 
+    subtitle="Share payment link via an email, SMS, messenger, chatbot etc." 
+    titlePrefix={<DollarIcon size="xl" />}
```

Comment:
One thing I notice while implementing the Card is that in the titlePrefix icon the user won't just need to pass `size` but also `color`

```jsx
      <CardComponent.Header
        title="Card Header"
        subtitle="Subtitle"
        titlePrefix={<AlertOctagonIcon color="surface.text.muted.lowContrast" size="xlarge" />}
        titleSuffix={<Counter value={12} />}
      />
```

Reply:
we have a restriction to use only the large icon but this is not easily achievable with the current API we are just relying on consumers

Reply:
Resolved with the new API.